mlflow/environment_variables.py

```diff
@@ -316,6 +316,18 @@ def get(self):
 # Specifies whether or not to use multipart upload when uploading large artifacts.
 MLFLOW_ENABLE_MULTIPART_UPLOAD = _BooleanEnvironmentVariable("MLFLOW_ENABLE_MULTIPART_UPLOAD", True)
 
+#: Specifies the timeout for multipart upload chunk requests.
+#: (default: ``None``)
+MLFLOW_MULTIPART_CHUNK_UPLOAD_TIMEOUT = _EnvironmentVariable(
```

Comment:
Would it be helpful to add the units to the name and the env var description? `SECONDS` and to mention that shorter timeouts are possible with fractional seconds definitions.

Reply:
great idea!

Reply:
/ai improve the comment based on Ben's suggestion above.